### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by scireumskip**
This pull request was initiated by Lokalise (user Jan Müller) at 2020-05-12 12:49:54

### CODE REVIEW

File: src/main/resources/biz_cs.properties [hunk 1]
```diff
@@ -50,36 +135,609 @@ Country.tr = Turecko
...
+DeleteProcessesTask.deleteCount = Nejsou k dispozici žádné procesy Proces je odstraněn ${count} procesy budou smazány.
+DeleteTenantJobFactory.aborting = Při mazání došlo k chybě, proces je zrušen.
+DeleteTenantJobFactory.deletingMainEntity = Smazat datový objekt klienta...
+DeleteTenantJobFactory.description = Odstraní vybraného klienta a všechna přidružená data.
+DeleteTenantJobFactory.label = Odstranit klienta
+DeleteTenantJobFactory.simulateInfo = Odstranění je pouze simulováno.
+DeleteTenantJobFactory.takeAction = Opravdu smazat?
+DeleteTenantJobFactory.takeAction.help = Pokud je vybrána možnost 'Ne', bude vymazání pouze simulováno.
+DeleteTenantJobFactory.title = Smazat klienta "${name}"
+DeleteTenantJobFactory.title.simulate = Smazat klienta "${name}" (SIMULATED)
```
Comment (bwiedmann) [inline src/main/resources/biz_cs.properties:157]:
```suggestion
DeleteTenantJobFactory.title.simulate = Smazat klienta '${name}' (SIMULATED)
```

File: src/main/resources/biz_cs.properties [hunk 2]
```diff
@@ -50,36 +135,609 @@ Country.tr = Turecko
...
+DateTimeFormatCheck.remark = Formát data: ${format}
+DeleteProcessesTask.deleteCount = Nejsou k dispozici žádné procesy Proces je odstraněn ${count} procesy budou smazány.
+DeleteTenantJobFactory.aborting = Při mazání došlo k chybě, proces je zrušen.
+DeleteTenantJobFactory.deletingMainEntity = Smazat datový objekt klienta...
+DeleteTenantJobFactory.description = Odstraní vybraného klienta a všechna přidružená data.
+DeleteTenantJobFactory.label = Odstranit klienta
+DeleteTenantJobFactory.simulateInfo = Odstranění je pouze simulováno.
+DeleteTenantJobFactory.takeAction = Opravdu smazat?
+DeleteTenantJobFactory.takeAction.help = Pokud je vybrána možnost 'Ne', bude vymazání pouze simulováno.
+DeleteTenantJobFactory.title = Smazat klienta "${name}"
```
Comment (bwiedmann) [inline src/main/resources/biz_cs.properties:156]:
```suggestion
DeleteTenantJobFactory.title = Smazat klienta '${name}'
```

File: src/main/resources/biz_fr.properties [hunk 3]
```diff
@@ -142,6 +142,7 @@ DatabaseController.reason = raison
 DatabaseController.schema = Schéma de la base de données
 DatabaseController.sql = SQL
 DatabaseController.unknownChange = Changement inconnu
+DateRangeParameter.label = Période
 DateTimeFormatCheck.errorMsg = La valeur "${value}" ne correspond pas au format de date "${format}".
```
Comment (bwiedmann) [inline src/main/resources/biz_fr.properties:146]:
```suggestion
DateTimeFormatCheck.errorMsg = La valeur '${value}' ne correspond pas au format de date '${format}'.
```

File: src/main/resources/biz_pl.properties [hunk 4]
```diff
@@ -142,6 +142,7 @@ DatabaseController.reason = Powód:
 DatabaseController.schema = Schemat bazy danych
 DatabaseController.sql = SQL
 DatabaseController.unknownChange = Nieznana zmiana
+DateRangeParameter.label = Okres
 DateTimeFormatCheck.errorMsg = Wartość "${value}" nie zgadza się z formatem daty "${format}".
```
Comment (bwiedmann) [inline src/main/resources/biz_pl.properties:146]:
```suggestion
DateTimeFormatCheck.errorMsg = Wartość '${value}' nie zgadza się z formatem daty '${format}'.
```

File: src/main/resources/biz_cs.properties (hunk 1 above)
Comment (jmuscireum) [reply src/main/resources/biz_cs.properties:157]: These are the same in all languages

File: src/main/resources/biz_cs.properties (hunk 2 above)
Comment (jmuscireum) [reply src/main/resources/biz_cs.properties:156]: These are the same in all languages

**Pull request merged**